recipes/miniaudio/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "0.10.32":
+    url: https://github.com/mackron/miniaudio/archive/d06d4983d3ff512690eb37f5edd25bf96f8f3764.tar.gz
```

Comment:
Upstream uses number for releases but no git tags 🤔

Reply:
I know, right? I think their goal is to always use latest master. All master commits correspond to a version, but yeah, for the sake of conan versioning, this is what we have to deal with.

Reply:
Than we should mayber use our virtual releases?